packages/table/src/Row/InternalRowWithoutRT.tsx

```diff
@@ -31,7 +31,9 @@ const InternalRowWithoutRT = ({
       {...rest}
     >
       {React.Children.map(children, (child: ReactNode, index: number) => {
-        return <Cell {...(child as ReactElement).props} cellIndex={index} />;
+        return (
+          <Cell key={`cell-${index}`} {...(child as ReactElement).props} />
```

Comment:
intentional to rm cellIndex prop?

Reply:
Yes. `cellIndex` is only used in `InternalCell`, so `InternalRowWithRT`